clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -2233,12 +2233,14 @@ llvm::Constant *CodeGenModule::EmitAnnotateAttr(llvm::GlobalValue *GV,
 
   llvm::Type *ResType = llvm::PointerType::getInt8PtrTy(
       this->getLLVMContext(), GV->getType()->getPointerAddressSpace());
-  llvm::Constant *C =
-      llvm::ConstantExpr::getPointerBitCastOrAddrSpaceCast(GV, ResType);
+
   // Create the ConstantStruct for the global annotation.
   llvm::Constant *Fields[4] = {
-      C, llvm::ConstantExpr::getBitCast(AnnoGV, Int8PtrTy),
-      llvm::ConstantExpr::getBitCast(UnitGV, Int8PtrTy), LineNoCst};
+    llvm::ConstantExpr::getPointerBitCastOrAddrSpaceCast(GV, ResType),
```

Comment:
This should be indented by 4 whitespaces (see https://llvm.org/docs/CodingStandards.html#braced-initializer-lists).